spec/features/create_user_spec.rb

```diff
@@ -34,4 +34,90 @@
       expect(page).to have_content "Password confirmation doesn't match Password"
     end
   end
-end
+
+  context "with invalid email" do
+
+    before do
+      visit "/users/new"
+      fill_in "Email", with: "nonsense"
+      fill_in "Password", with: "Password"
+      fill_in "Password confirmation", with: "Password"
+      click_button "Submit"
+    end
+
+    it "tells the user they have signed up" do
+      expect(page).to have_content "You have signed up"
+    end
+  end
+
+  context "sign in with Twitter" do
+
+    before do
+      visit "/users/new"
+      click_link "Sign in with Twitter"
+    end
+
+    it "leads to Twitter authentication page" do
+      expect(current_path).to eq "/auth/twitter"
```

Comment:
Because of the twitter-omniauth gem, this test will NEVER pass! Be sure to run your tests before you make your Pull Request!  All your tests need to pass!